models/uuv_hippocampus/uuv_hippocampus.sdf

```diff
@@ -0,0 +1,645 @@
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='back_left_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_1_joint</jointName>
+  <linkName>rotor_1</linkName>
+  <turningDirection>cw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='front_left_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_2_joint</jointName>
+  <linkName>rotor_2</linkName>
+  <turningDirection>ccw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='back_right_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_3_joint</jointName>
+  <linkName>rotor_3</linkName>
+  <turningDirection>cw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>-->
+
+    <plugin name="gps_plugin" filename="libgazebo_gps_plugin.so">
+        <robotNamespace/>
+        <gpsNoise>true</gpsNoise>
+    </plugin>
```

Comment:
Does the hippocampus have GPS?

Reply:
Our real world version does not have GPS since GPS does not work/is attenuated almost immediately underwater. Underwater localization is some kind of a night mare which I would prefer for the moment to keep out of the simulation
Hence we implemented "GPS" as a well integrated work around with fairly similar characteristics. However, this can be of course extended at some point but it will be quite a chunk of work to model the real world localization characteristics.

Reply:
Fair enough. I don't think we need to "fix" this, but still be aware the state estimation is using GPS data